document/CoreFormat.mdk

```diff
@@ -413,6 +413,99 @@ The `results` array shall be empty if the run that produced the `run` object did
 If the tool was run solely for the purpose of exporting rule metadata (see &sect;[#run-rules]),
 the `results` property shall be absent.
 
+### `toolNotifications` { #run-toolNotifications }
+
+A `run` object may contain a property named `toolNotifications`
+whose value is an array of zero or more `notification` objects (&sect;[#notification]).
+Each element of the array represents a runtime condition detected by the tool.
+The presence within this array of any `notification` object whose `level` property (&sect;[#notification-level]) is `error` shall mean
+that the run failed.
+
+~ IsoNumberedNote { @IsoNumberedNote=1 }
+The information in `toolNotifications` is primarily intended for the developers of the analysis tool,
```

Comment:
Yes, this is the primary audience, with one additional note that viewers/automation systems will minimally want to consult these sections to ensure they don't contain errors (and to communicate that fact to users if they do). Makes sense? the content/detail in the sections are targeted towards the tool owner, but the results consumer, minimally, needs to be alerted that this content exists (particularly for errors).